serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/protobuf/ProtobufSerializer.java

```diff
@@ -25,7 +25,8 @@
 import com.google.protobuf.Descriptors;
 import com.google.protobuf.Message;
 import com.squareup.wire.schema.internal.parser.ProtoFileElement;
-import io.apicurio.registry.utils.protobuf.schema.FileDescriptorUtils;
+import com.amazonaws.services.schemaregistry.utils.apicurio.FileDescriptorUtils;
+//import io.apicurio.registry.utils.protobuf.schema.FileDescriptorUtils;
```

Comment:
Is this intentional? Please add a TODO if this is tech debt.

Reply:
Same as above. I will address in the next PR.

Reply:
I just left in the commented out import since I'm assuming we are going to patch back the real apicurio soon - should I remove it?

Reply:
We can fix it later, please add a TODO